exercises/I-array-properties/exercise.js

```diff
@@ -5,9 +5,19 @@
 
 */
 
-function isEmpty(arr) {
-  return; // complete this statement
-}
+// function isEmpty(arr) {
+//    if (arr === 0) {
```

Comment:
Dear Maira, nice job! 

In this line of code we are referring to the length of the array not the value so it should be ;
unction isEmpty(arr) {
  if (arr.length === 0) {
   return true;
  }
  return false; 
}